js/darkmode.js

```diff
@@ -0,0 +1,43 @@
+const icon = document.getElementById('icon');
+onload = onload();
+icon.onclick = function () {
+  const element = document.body;
+  element.classList.toggle('dark-theme');
+  if (element.classList.contains('dark-theme')) {
+    icon.src = 'img/icons/sun.png';
+    setCookie('theme', 'dark', 30);
+  } else {
+    icon.src = 'img/icons/moon.png';
+    setCookie('theme', 'light', 30);
+  }
+};
+
+function onload() {
+  if (getCookie('theme') === 'light') {
```

Comment:
What if the cookie is not present before? It will show dark mode!!🤔

Reply:
Yes It is showing dark mode as default . Shall I change it to light Mode as default ?

Reply:
Yes, light mode is default.